src/routes.ts

```diff
@@ -34,8 +39,32 @@ const routes: Routes = [
     module: ReportWorkspaceModule,
   },
   {
-    path: '/error-suppressions',
-    module: ErrorSuppressionsModule,
+    path: '/admin',
+    module: AdminModule,
+    children: [
+      {
+        path: '/qa-certification',
+        module: QaMaintenanceModule,
+        children: [
+          {
+            path: '/test-data',
```

Comment:
```suggestion
            path: '/test-summaries',
```

Reply:
or `/tests` maybe

Reply:
/test-data is coming from the requirements we agreed on yesterday and it's in the ticket so I'd rather keep it the way it's in in the ticket.

Reply:
Based on our discussion i'll change to qa-test-summary for the endpoint